src/test-dummies/cmd_utils/bin/pgrep

```diff
@@ -0,0 +1 @@
+spawned_mock
```

Comment:
Shouldn't we be removing the file for 'pidof' if is it not used anymore?

Reply:
Yes, we don't need it.